File: examples/java/sdk-reference/error-handling/retry-strategy-config-signature.java
RetryStrategy RetryStrategies.exponentialBackoff(
    int maxAttempts,
    Duration initialDelay,
    Duration maxDelay,
    double backoffRate,
    JitterStrategy jitter
)

RetryStrategy RetryStrategies.fixedDelay(
    int maxAttempts,
    Duration fixedDelay
)

File: examples/csharp/sdk-reference/error-handling/retry-strategy-config-signature.cs
// .NET has no config object; build strategies with the RetryStrategy factory.
public static IRetryStrategy RetryStrategy.Exponential(
    int maxAttempts = 3,
    TimeSpan? initialDelay = null,          // default 5s
    TimeSpan? maxDelay = null,              // default 300s
    double backoffRate = 2.0,
    JitterStrategy jitter = JitterStrategy.Full,
    Type[]? retryableExceptions = null,
    string[]? retryableMessagePatterns = null);
